### DISCUSSION TITLE
Some confusion for me

### DISCUSSION THREAD

**Discussion by jeromeleong** [XAMPPRocky/fluent-templates#46] [Q&A]
I am a beginner in Rust and I want to know how to use a specified FTL file. 
For example, en-US has main.ftl and other.ftl, but I only want to read the content of other.ftl.

**Comment by XAMPPRocky** [comment]
Thank you for your issue! In this crate all FTL files of a locale are put into a single bundle, so you always have all the definitions `main.ftl` and `other.ftl` available in the loader, there's no way to have only one file available without making it a different locale. Usually you try to keep the message IDs unique in each file as that makes it possible to use tools like Mozilla's Pontoon for message ID tracking.
